AutomationHelpers/src/UserCodeCollections/StringLibrary.cs

```diff
@@ -24,5 +24,29 @@ public static string ConcatStrings(string value1, string value2)
         {
             return String.Concat(value1, value2);
         }
+
+        /// <summary>
+        /// Creates and returns a random string. If a string shorter than 35 characters is used it may not be unique.

```

Comment:
"If a string shorter than 35 characters is used it may not be unique." Why's the threshold at 35? Are strings >=35 chars always unique? No, right?
Remove here. Add rewritten info in param, unless that makes it too long. If so, keep rewritten info in summary.